conf/machine/imx51evk.conf

```diff
@@ -9,9 +9,13 @@ MACHINEOVERRIDES =. "mx5:mx51:"
 include conf/machine/include/imx-base.inc
 include conf/machine/include/tune-cortexa8.inc
 
-# Disable perf features as 2.6.35.3 fails to build otherwise
-PERF_FEATURES_ENABLE = ""
-
 KERNEL_DEVICETREE = "imx51-babbage.dtb"
 
+# This machine is not supported by u-boot-imx as it is not tested by NXP on this
+# board. So we force it to use u-boot-fslc which is based on mainline here.
+IMX_DEFAULT_BOOTLOADER = "u-boot-fslc"
+
+UBOOT_MAKE_TARGET = "u-boot.imx"
+UBOOT_SUFFIX = "imx"
```

Comment:
These are correct for `u-boot-fslc`?

Reply:
Those are; this does not use SPL so the image file is `u-boot.imx`.